Services/sources/TMU.c

```diff
@@ -0,0 +1,544 @@
+/*
+ * TMU.c
+ *
+ * Created: 2/24/2020 11:47:28 AM
+ *  Author: Reem
+ */
+#include "TMU.h"
+
+/*******Static Global Variables*********/
+static TMU_Cfg gstr_TMUcfg;
```

Comment:
Init all the global varabiles in the init function and in deinit function as well